src/utils/object.ts

```diff
@@ -49,11 +31,21 @@ export function recursiveObjectSort(object: object): object {
 }
 
 /**
- * lodash cloneDeep only clones enumerable properties
+ * Deep clone of object
```

Comment:
Do you think it might be useful to delegate this to, say, [cloner](https://www.npmjs.com/package/cloner)?